txpipe/twopoint.py

```diff
@@ -250,12 +253,14 @@ def add_data_points(self, S, results):
         GAMMAT = sacc.standard_types.galaxy_shearDensity_xi_t
         GAMMAX = sacc.standard_types.galaxy_shearDensity_xi_x
         WTHETA = sacc.standard_types.galaxy_density_xi
+        POS_EXT = "pos_ext"
+        SHEAR_EXT = "shear_ext"
 
         comb = []
         for index, d in enumerate(results):
             # First the tracers and generic tags
-            tracer1 = f"source_{d.i}" if d.corr_type in [XI, GAMMAT] else f"lens_{d.i}"
-            tracer2 = f"source_{d.j}" if d.corr_type in [XI] else f"lens_{d.j}"
+            tracer1 = f"source_{d.i}" if d.corr_type in [XI, GAMMAT, SHEAR_EXT] else f"lens_{d.i}"
+            tracer2 = f"source_{d.j}" if d.corr_type in [XI] else (f"external_{d.j}" if d.corr_type in [POS_EXT, SHEAR_EXT] else f"lens_{d.j}")
```

Comment:
does this way of writing an `elif` actually work?

Reply:
i think it worked, but probably wasn't very clear. I made it a longer set of if and elif statements